docs/pages/cli.mdx

```diff
@@ -32,11 +33,39 @@ The valid templates are:
 
 **`minimal`**: A minimal template with javascript for the frontend (no framework)
 
+## `dev-contracts`
+
+Start a local MUD development server.
+
+The tool with start a local Ethereum node in the background, or use a custom node via the provided `--rpc` argument.
+
+It will run all the necessary codegeneration steps, deploy the MUD contracts to the local node.
+It will also watch for file changes to the `mud.config.ts` file and all Solidity source files, and re-run the codegen or deploy steps if necessary.
+The latest deployment is saved to the `worlds.json` file, where the client can find it to always connect to the latest local deployment.
+
+The `tsgenOutput` argument is used to specify the output path for code generated for `recs`.
+
+To run the dev server with internal development node:
+
+```bash
+pnpm mud dev-contracts --tsgenOutput ../client/src/mud
+```
+
+To run the dev server with a custom development node:
+
+```bash
+# Terminal 1:
+anvil --port 8545
+
+# Terminal 2:
+pnpm mud dev-contracts --tsgenOutput ../client/src/mud --rpc localhost:8545
+```
```

Comment:
Maybe a note at the end of this section about the anvil port needing to be adjusted in client if customized?

Reply:
this doc is technically unrelated to the templates, wonder if it would be best to couple it with the templates? But also fine to mention it here too